### DISCUSSION TITLE
Set UnlimitedLengthInDocumentLibrary property on multi-line field

### DISCUSSION THREAD

**Discussion by hoppalazi** [pnp/pnpcore#1587] [Ideas]
Is there a way to set the property [UnlimitedLengthInDocumentLibrary](https://learn.microsoft.com/en-us/dotnet/api/microsoft.sharepoint.client.fieldmultilinetext.unlimitedlengthindocumentlibrary?view=sharepoint-csom) on a multi-line field?

**Comment by jansenbe** [comment]
Did you consider crafting the Field schema XML yourselves and create the field using https://pnp.github.io/pnpcore/using-the-sdk/fields-add.html#add-field-via-field-xml
